website/release_announcement_drafts/v1.2.0.md

```diff
@@ -0,0 +1,33 @@
+We're excited to announce the v1.2.0 release of JBrowse Web!
+
+## New plugin store added to website
+
+One of the core aspects of JBrowse 2 is that it is an extensible
+platform for biological visualization that can be extended with
+plugins. We are excited to introduce the first version of our
+plugin store, where we list the current external plugins that
+are available. Check it out [here](https://jbrowse.org/jb2/plugin_store).
+In the coming weeks, we will also be bringing this plugin store directly
+into the application, allowing plugin installation with the click of a button.
+Stay tuned!
+
+## Native SVG export
+
+We're excited to introduce a new feature to JBrowse Web: built-in
+SVG export of track visualizations!
+
+![](./img/svg_screenshot.png)
```

Comment:
could consider this screenshot, i like that it shows bigwig and human data for example https://github.com/cmdcolin/jb2export/blob/master/img/1.png

Reply:
Replaced the existing with this one, nice 😎